otx/mpa/det/trainer.py

```diff
@@ -56,7 +56,8 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         datasets = [build_dataset(cfg.data.train)]
 
         # FIXME: Currently detection do not support multi batch evaluation. This will be fixed
-        cfg.data.val_dataloader.samples_per_gpu = 1
+        if 'val' in cfg.data:
+            cfg.data.val_dataloader.samples_per_gpu = 1
 
         # FIXME: scale_factors is fixed at 1 even batch_size > 1 in simple_test_mask
         # Need to investigate, possibly due to OpenVINO
```

Comment:
You missed this block! Please include it as well. :)

Reply:
I'm going to remove line 62-66 because if there is a validation dataset, `val_dataloader.samplers_per_gpu` will be set to 1.

Reply:
What do you think?

Reply:
Yes, then no object on that. I also left this part for the reference because there is a `FIXME` I didn't consider at all.
If this `FIXME` is considered `FIXED`, then I believe we can safely get rid of it.

Reply:
Actually, in the previous version, there is no problem to use `val_dataloader.samplers_per_gpu` > 1 for detection, classification except for instance segmentation, semantic segmentation. However, I'm also not sure the latest mmdetection version. So, let's keep as is with `FIXME` tag.